### ISSUE TITLE
Append to a named stash by default

### ISSUE THREAD

**Issue by codebykat** [josh-berry/tab-stash#110]
### Problem Statement

For easier organizing of tabs, it would be very cool to be able to specify a default named stash (such as "inbox" or "unsorted") rather than the "Append to the top-most stash" option. Right now even with a high number in the "minutes" option, it creates a large number of stashes.

I can sort of hack this by putting a ridiculously high number for "minutes", except that if the top-most stash is named, it still creates a new one, so I can't have an "inbox" stash.

It would be very helpful to have all unsorted tabs stashed in a single place so that I know I have not yet organized them.

### Preferred Solution(s)

Add a setting to append to a named stash by default.

### Alternative Solution(s)

Have a setting to "always append to the top-most stash" regardless of the number of minutes elapsed or whether the top-most stash is named.

### Additional Context

Thank you for your work on this add-on, it always works so smoothly and is a huge help in keeping me organized!

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
Definitely, I think having some type of inbox is a great idea. I'm an avid follower of Getting Things Done so I completely understand the need for an "inbox" stash, especially for situations where you have a bunch of open tabs you want to save mixed in with a bunch of stuff you don't need (which happens to me often...). I'll keep this in mind!

> Thank you for your work on this add-on, it always works so smoothly and is a huge help in keeping me organized!

My pleasure! I'm really glad to hear you've found it useful.

**Comment by klhavlik** [comment]
It would be nice to have a stash for notes or to-do lists, something that you could edit and pin (it would be awesome if it had full text editing features).  I use the internet a LOT more frequently than I check email and I can't find an email client that efficiently handles notes/to-do lists.  I haven't been able to find a decent "notes" app or extension either and I think that a stash tab could possibly serve that function well.  I realize that it's a bit of a different scope but just thought I'd put it out there!

I am really enjoying "tab stash".  I usually keep tons of tabs up when using Firefox.  I should probably bookmark them but I really HATE Windows' built-in bookmarking...not very flexible IMHO. Tab Stash works great for that!

**Comment by josh-berry** [comment]
> It would be nice to have a stash for notes or to-do lists, something that you could edit and pin (it would be awesome if it had full text editing features). ...

Yeah I agree this is something that should exist. You're right that it's a bit out of scope for Tab Stash, but it's definitely something I've been thinking about.

**Comment by josh-berry** [comment]
Noting here that in #530, a couple folks have asked for saving tabs to different groups based on rules (e.g. certain sites always go to certain folders, creating folders per day, etc.).
